hydrolib/core/io/rr/models.py

```diff
@@ -514,7 +514,7 @@ class RainfallRunoffModel(ParsableFileModel):
     @classmethod
     def property_keys(cls) -> Iterable[str]:
         # Skip first element corresponding with file_path introduced by the FileModel.
```

Comment:
Perhaps update the comment why we are now skipping the first 2?

Reply:
I have updated the comment.